File: A-I_Scream.go
package main
import (
	"fmt"
)
func main() {
	var a, b int
	fmt.Scan(&a, &b)
	var c, ans int
	c = a + b

	if c >= 15 && b >= 8 {
		ans = 1
	} else if c >= 10 && b >= 3 {
		ans = 2
	} else if c >= 3 {
		ans = 3
	} else {
		ans = 4
	}

	fmt.Println(ans)
}

File: A-I_Scream.cpp
#include <iostream>
using namespace std;

int judgment(int a, int b){
    int c = a + b;
    if ((c) >= 15 && b >= 8){
        return 1;
    } else if (c >= 10 && b >= 3){
        return 2;
    } else if (c >= 3){
        return 3;
    } else{
        return 4;
    } 

}

int main(){
    int a, b, ans;
    cin >> a >> b;
    ans = judgment(a, b);
    cout << ans << endl;
    


}